scripts/rollup/cjs-check-plugin.mjs

```diff
@@ -9,13 +9,14 @@ function cleanup(opts = {}) {
   return {
     name: "cjs-check",
 
-    renderChunk(code, chunk) {
+    async renderChunk(code, chunk) {
       if (opts.extension !== '.js') {
         return null;
       } else if (!filter(chunk.fileName)) {
         return null;
       }
 
+      await init()
```

Comment:
Looks like when using `cjs-module-lexer` in ESM it needs us to do an `async` init